10/alpine/Dockerfile

```diff
@@ -37,14 +42,27 @@ RUN addgroup -g 1000 node \
     && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
     && grep " node-v$NODE_VERSION.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
     && tar -xf "node-v$NODE_VERSION.tar.xz" \
+    && ln -s /usr/bin/ccache /usr/local/bin/gcc \
+    && ln -s /usr/bin/ccache /usr/local/bin/g++ \
```

Comment:
Couldn't these be set via `export CC=... CXX=...`?  Or does the Node.js build system not support those properly?

Reply:
Not sure what you mean? I was following the ccache instructions on installation